### DISCUSSION TITLE
Minimal Mistakes remote theme starter

### DISCUSSION THREAD

**Discussion by wangyi111** [mmistakes/minimal-mistakes#2645] [Q&A] [answered]
I am new to jekyll so I followed the quick start documentation to create a test site using the [Minimal Mistakes remote theme starter](https://github.com/mmistakes/mm-github-pages-starter/generate), which uses a remote theme. Things went well until I wanted to create new navigation classes (e.g. "publications"). It seems impossible to generate a "publication" page which should look similar to "post" page, as I copied the structure of "posts" (including create a folder for publication markdown files) but the url is always linked to empty page. If possible please have a look at [my repository](https://github.com/wangyi111/minimal-web-trial). Should I try other ways instead of using the simple generated "remote theme starter"?

**Comment by mmistakes** [comment, marked as answer]
Have a look at Jekyll's documentation. You created a new collection `_publications`, but you never configured it.

https://jekyllrb.com/docs/collections/

**Comment by wangyi111** [comment]
Thank you! It works now!
